### PR TITLE
Training UX + Boolean Value

### PR DESCRIPTION
**Pull request by eyeshoe**

### CODE REVIEW

File: app/src/main/java/org/sagebase/crf/step/CrfSkipMCStepLayout.java [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+    @Override
+    protected void refreshCrfSubmitBar() {
+        super.refreshCrfSubmitBar();
+
+        crfNextButton.setBackgroundResource(R.drawable.crf_rounded_button_gray);
+        crfNextButton.setTextColor(ResourcesCompat.getColor(getResources(), R.color.rsb_white, null));
+        crfNextButton.setText("Done");
+        crfBackButton.setVisibility(GONE);
+
+        crf_title.setTextColor(Color.rgb(255, 255, 255));
```
Comment (nategbrown9) [inline app/src/main/java/org/sagebase/crf/step/CrfSkipMCStepLayout.java]: Can just use Color.WHITE

File: app/src/main/java/org/sagebase/crf/step/CrfInstructionStepLayout.java [hunk 2]
```diff
@@ -210,4 +246,22 @@ public boolean crfToolbarShowProgress() {
     public boolean controlMediaVolume() {
         return crfInstructionStep.mediaVolume;
     }
+
+    public void remindMeLater() {
```
Comment (nategbrown9) [inline app/src/main/java/org/sagebase/crf/step/CrfInstructionStepLayout.java]: I don't think the reminders will actually get scheduled without handling the activity result.  Look at CrfStartTaskStepLayout for an example, or could CrfSkipInstructionStepLayout just extend CrfStartTaskStepLayout?

File: app/src/main/java/org/sagebase/crf/step/CrfSkipMCStepLayout.java [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        this.crf_text = findViewById(R.id.rsb_survey_text);
+        this.remindMeLaterButton = findViewById(R.id.crf_submit_bar_skip);
+
+    }
+
+    @Override
+    protected void initStepLayout(FormStep step)
+    {
+        super.initStepLayout(step);
+        container.setBackgroundColor(Color.parseColor("#C94281"));
```
Comment (nategbrown9) [inline app/src/main/java/org/sagebase/crf/step/CrfSkipMCStepLayout.java]: Please specify color in colors.xml.

File: .idea/assetWizardSettings.xml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (nategbrown9) [inline .idea/assetWizardSettings.xml]: Should probably remove this file and add it to .gitignore.

File: app/src/main/java/org/sagebase/crf/step/CrfInstructionStep.java [hunk 5]
```diff
@@ -74,6 +78,12 @@
      */
     public boolean mediaVolume;
 
+    /**
+     * If true, the remind me later button will show, if false it will hide, defaults to true
+     */
+    public boolean remindMeLater;
```
Comment (nategbrown9) [inline app/src/main/java/org/sagebase/crf/step/CrfInstructionStep.java]: Should this move with the rest of the reminder stuff to CrfSkipInstructionStep?

File: app/src/main/java/org/sagebase/crf/step/CrfInstructionStepLayout.java [hunk 6]
```diff
@@ -48,6 +61,7 @@
     protected View rootInstructionLayout;
     protected ImageButton customButton;
     protected TextView customButtonText;
+    protected Button remindMeLaterButton;
```
Comment (nategbrown9) [inline app/src/main/java/org/sagebase/crf/step/CrfInstructionStepLayout.java]: Can this move with the handling code to CrfSkipInstructionStepLayout?

File: app/src/main/java/org/sagebase/crf/step/CrfInstructionSurveyItem.java [hunk 7]
```diff
@@ -81,4 +81,8 @@
      */
     @SerializedName("mediaVolume")
     public boolean mediaVolume;
+
+
+    @SerializedName("remindMeLater")
```
Comment (nategbrown9) [inline app/src/main/java/org/sagebase/crf/step/CrfInstructionSurveyItem.java]: Can this move to CrfSkipInstructionSurveyItem?

File: app/src/main/java/org/sagebase/crf/step/active/HeartBeatSample.java [hunk 8]
```diff
@@ -57,4 +57,26 @@ public String toString() {
...
+    public boolean isPressureExcessive() {
+        int hr = bpm;
+
+        if(hr > 250 || hr <= 40) {
+            return true;
+        }
+
+        // for testing purposes
+        return true;
+        //return false;
```
Comment (nategbrown9) [inline app/src/main/java/org/sagebase/crf/step/active/HeartBeatSample.java]: Should this be false by default?

File: app/src/main/java/org/sagebase/crf/step/active/HeartBeatSample.java [hunk 9]
```diff
@@ -57,4 +57,26 @@ public String toString() {
...
+    }
+
+    // Algorithm to be implemented
+    public boolean abnormalHR() {
+        return true;
```
Comment (nategbrown9) [inline app/src/main/java/org/sagebase/crf/step/active/HeartBeatSample.java]: Should this be false by default?

File: app/src/main/java/org/sagebase/crf/step/active/HeartBeatSample.java [hunk 10]
```diff
@@ -57,4 +57,26 @@ public String toString() {
...
+    }
+
+    // Algorithm to be implemented
+    public boolean declineHR() {
+        return true;
```
Comment (nategbrown9) [inline app/src/main/java/org/sagebase/crf/step/active/HeartBeatSample.java]: Should this be false by default?

File: app/src/main/java/org/sagebase/crf/step/active/HeartBeatSample.java (hunk 8 above)
Comment (eyeshoe) [reply app/src/main/java/org/sagebase/crf/step/active/HeartBeatSample.java]: Yes, but for testing purposes I kept it as true

File: app/src/main/java/org/sagebase/crf/step/active/HeartBeatSample.java (hunk 9 above)
Comment (eyeshoe) [reply app/src/main/java/org/sagebase/crf/step/active/HeartBeatSample.java]: Yes, but for testing purposes I kept it as true

File: app/src/main/java/org/sagebase/crf/step/active/HeartBeatSample.java (hunk 10 above)
Comment (eyeshoe) [reply app/src/main/java/org/sagebase/crf/step/active/HeartBeatSample.java]: Yes, but for testing purposes I kept it as true

File: app/src/main/java/org/sagebase/crf/step/CrfInstructionStep.java (hunk 5 above)
Comment (eyeshoe) [reply app/src/main/java/org/sagebase/crf/step/CrfInstructionStep.java]: Just moved it over!

File: app/src/main/java/org/sagebase/crf/step/CrfInstructionSurveyItem.java (hunk 7 above)
Comment (eyeshoe) [reply app/src/main/java/org/sagebase/crf/step/CrfInstructionSurveyItem.java]: Moved

File: app/src/main/java/org/sagebase/crf/step/CrfInstructionStep.java (hunk 5 above)
Comment (eyeshoe) [reply app/src/main/java/org/sagebase/crf/step/CrfInstructionStep.java]: Moved

File: app/src/main/java/org/sagebase/crf/step/CrfInstructionStepLayout.java (hunk 6 above)
Comment (eyeshoe) [reply app/src/main/java/org/sagebase/crf/step/CrfInstructionStepLayout.java]: Moved

**Pull request closed without merging**